core/src/autogluon/core/utils/files.py

```diff
@@ -115,13 +115,14 @@ def check_sha1(filename, sha1_hash):
 def mkdir(path):
     """Make directory at the specified local path with special error handling.
     """
-    try:
-        os.makedirs(path)
-    except OSError as exc:  # Python >2.5
-        if exc.errno == errno.EEXIST and os.path.isdir(path):
-            pass
-        else:
-            raise
+    if len(path) > 0:
```

Comment:
what if hypothetically the working directory doesn't exist? This doesn't need to be addressed but I see this as the weird edge-case that might not have been considered.